distribution/war/src/main/webapp/resources/uicomponents/widgets/extraGeneVariantData.js

```diff
@@ -191,7 +191,7 @@ var ExtraGeneVariantData = (function (ExtraGeneVariantData) {
       var hintTool = newButtonRow.down('span.fa.fa-question-circle.xHelpButton');
       var icon_helpController = new PhenoTips.widgets.HelpButton(hintTool);
 
-      Event.fire(document, 'xwiki:dom:updated', {elements :[newGeneRow]});
+      Event.fire(document, 'xwiki:dom:updated', {elements :[newGeneRow, newMoreInfoRow]});
```

Comment:
Yes, thanks for pointing it out, @teyden just noticed it too. This is a mistake, I pasted the change into the wrong method. I'm about to push a fix.